tests/sentry/mail/test_utils.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from uuid import uuid4
+
+from sentry.mail.utils import mail_action, migrate_project_to_issue_alert_targeting
+from sentry.models import Rule, UserOption
+from sentry.testutils import TestCase
+from sentry.plugins.base import plugins
+
+
+class MigrateProjectToIssueAlertTargetingTest(TestCase):
+    def setUp(self):
+        self.organization = self.create_organization()
+        self.user = self.create_user()
+        self.user_2 = self.create_user()
+        self.team = self.create_team(self.organization, members=[self.user, self.user_2])
+        self.project = self.create_project(organization=self.organization, teams=[self.team])
+        self.mail = plugins.get("mail")
+
+    def create_rule(self, actions):
+        data = {
+            "conditions": [
+                {"id": "sentry.rules.conditions.first_seen_event.FirstSeenEventCondition"}
+            ],
+            "actions": actions,
+        }
+        return Rule.objects.create(project=self.project, label=uuid4().hex, data=data)
+
+    def assert_rule_actions_equal(self, rule, expected_actions):
+        rule_actions = rule.data.get("actions")
+        key = lambda action: action["id"]
+        assert rule_actions.sort(key=key) == expected_actions.sort(key=key)
```

Comment:
OMG, this doesn't test anything, since `.sort` just sorts in place and doesn't return anything 🤦